homeassistant/components/essent/sensor.py

```diff
@@ -36,6 +36,12 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                 tariff,
                 data['values']['LVR'][tariff]['unit']))
 
+    if not meters:
+        raise Exception(
```

Comment:
If you want the user to know, create a persistent notification instead.

Reply:
I'm not sure how I'd do that inside a component. It also doesn't seem to be normal for components to do so? (At least, I've never seen a component use notifications to me before) Still, I'd be willing to implement it that way if you could give me some pointers on how to do that.

Reply:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/google/__init__.py#L133-L136

Reply:
Done :)